sympy/polys/matrices/domainmatrix.py

```diff
@@ -57,13 +88,61 @@ def get_domain(cls, items_sympy, **kwargs):
         return K, items_K
 
     def convert_to(self, K):
+        r"""
+            convert_to: Returns a DomainMatrix with the desired domain or field
+
+            >>> from sympy import Matrix, CC
+            >>> from sympy.polys.matrices import DomainMatrix
+            >>> matrix = Matrix([[99, 16, 30, 62], [65, 79, 94, 15], [38, 13, 31, 47], [17, 2, 53, 78]])
+            >>> domainMatrix = DomainMatrix.from_Matrix(matrix)
+            >>> domainMatrix
+            DomainMatrix([[99, 16, 30, 62], [65, 79, 94, 15], [38, 13, 31, 47], [17, 2, 53, 78]], (4, 4), ZZ)
+
+            >>> domainMatrix.convert_to(CC)
+            DomainMatrix([[(99.0 + 0.0j), (16.0 + 0.0j), (30.0 + 0.0j), (62.0 + 0.0j)], [(65.0 + 0.0j), (79.0 + 0.0j), (94.0 + 0.0j), (15.0 + 0.0j)],
+            [(38.0 + 0.0j), (13.0 + 0.0j), (31.0 + 0.0j), (47.0 + 0.0j)], [(17.0 + 0.0j), (2.0 + 0.0j), (53.0 + 0.0j), (78.0 + 0.0j)]], (4, 4), CC)
+
+        """
         return self.from_rep(self.rep.convert_to(K))
 
     def to_field(self):
+        r"""
+            to_field: Returns a DomainMatrix with the appropriate field
+
+            >>> from sympy import Matrix
+            >>> from sympy.polys.matrices import DomainMatrix
+            >>> matrix = Matrix([[99, 16, 30, 62], [65, 79, 94, 15], [38, 13, 31, 47], [17, 2, 53, 78]])
+            >>> domainMatrix = DomainMatrix.from_Matrix(matrix)
+            >>> domainMatrix
+            DomainMatrix([[99, 16, 30, 62], [65, 79, 94, 15], [38, 13, 31, 47], [17, 2, 53, 78]], (4, 4), ZZ)
+
+            >>> domainMatrix.to_field()
+            DomainMatrix([[99, 16, 30, 62], [65, 79, 94, 15], [38, 13, 31, 47], [17, 2, 53, 78]], (4, 4), QQ)
+
+        """
         K = self.domain.get_field()
         return self.convert_to(K)
 
     def unify(self, other):
+        r"""
+            unify: Unifies any two DomainMatrix with different domains
+
+            >>> from sympy import Matrix
+            >>> from sympy.polys.matrices import DomainMatrix
+            >>> matrix = Matrix([[99, 16, 30, 62], [65, 79, 94, 15], [38, 13, 31, 47], [17, 2, 53, 78]])
+            >>> domainMatrix = DomainMatrix.from_Matrix(matrix)
+            >>> domainMatrix
+            DomainMatrix([[99, 16, 30, 62], [65, 79, 94, 15], [38, 13, 31, 47], [17, 2, 53, 78]], (4, 4), ZZ)
+            >>> Matrix1 = Matrix([[97, 36, 48, 61], [7, 38, 9, 69], [4, 29, 63, 21], [44, 84, 40, 19]])
+            >>> domainMatrix1 = DomainMatrix.from_Matrix(Matrix1)
```

Comment:
These examples are too dense and too long. It isn't necessary to use 4x4 matrices to illustrate this point. Use the smallest size that makes it clear what it happening e.g. 2x2.

Reply:
Thank you for the suggestion, it looks much more readable now.